doc/decisions/0007-feature-toggles-development.md

```diff
@@ -0,0 +1,92 @@
+# 0007. Feature toggles development
+
+| Date | Tags |
+|---|---|
+| 2018-05-25 | development |
+
+
+## Status
+
+Proposed
+
+## Context
+
+This change describes a methodology of development which we believe will help
+us increase the confidence and speed in the development process.
+
+### What problem are we trying to solve
+
+Currently the time between a PR is sent and the code gets merged is between 1 day
+to in some case 4 to 5 days.
```

Comment:
My experience is even worse, multiple larger PRs had 1-2weeks open-merge duration. It's caused by the fact that when PR author is honest and correctly states than some functionality requires full regression testing (and let's be realistic, it's almost always the case in any "deeper" changes down in the model/event handling), it takes multiple hours to test and testers are already overloaded, 2-3 obscure regressions are enough to push the open-merge duration to one week.